package.json

```
@@ -1,7 +1,7 @@
 {
-  "name": "ts-template",
+  "name": "@ubiquibot/command-start-stop",
   "version": "1.0.0",
-  "description": "Template repository with TypeScript support.",
+  "description": "Enables the assignment of tasks to hunters and the graceful unassignment.",
```

```suggestion
  "description": "Enables the assignment of tasks to contributors and the graceful unassignment.",
```

No more bounty hunter vernacular.